src/api/Config/types.ts

```diff
@@ -182,6 +182,7 @@ export interface INeo4jConfig extends IGraphVendorConfig {
   alternativeNodeId?: string;
   alternativeEdgeId?: string;
   allowVirtualEntities?: boolean;
+  timestampPropertyName?: string;
```

Comment:
this might move to the `IGraphVendorConfig` soon, but let's keep it here for now